features/step_definitions/assertion_steps.rb

```diff
@@ -2,6 +2,14 @@
   expect(page).to have_content content
 end
 
+Then("I should see a navigation bar") do
+  find('.nav_bar').visible?
+end
+
+Then("I should see {string} link") do |link|
+  page.should have_link(link)
```

Comment:
keep the syntax consistent
e.g
`expect(page)`

Reply:
done